order-api/src/main/java/com/example/orderapi/kafka/OrderResultConsumer.java

```diff
@@ -0,0 +1,30 @@
+package com.example.orderapi.kafka;
+
+import com.example.kafka.Event;
+import com.example.kafka.OrderCompleteEvent;
+import com.example.orderapi.application.service.OrderService;
+import com.example.orderapi.global.config.TopicNames;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@RequiredArgsConstructor
+@Component
+public class OrderResultConsumer {
+
+	private final OrderService orderService;
+
+	@KafkaListener(topics = TopicNames.ORDER_COMPLETE_TOPIC, groupId = "order-group")
```

Comment:
```suggestion
	@KafkaListener(
            topics = "${order.kafka.topic}",
            groupId = "${order.kafka.group-id}"
        )
```

```yml
order:
  kafka:
    topic: order-complete
    group-id: order-group
```

유지보수를 위해 이런 방법은 어떨까요?

Reply:
넵!! 수정 하도록 하겠습니다.